src/main/java/com/cburch/logisim/fpga/hdlgenerator/LedArrayColumnScanningHDLGeneratorFactory.java

```diff
@@ -85,14 +86,14 @@
     }
     return map;
   }
-  
+
   public static ArrayList<String> getPortMap(int id) {
     var map = new ArrayList<String>();
     if (HDL.isVHDL()) {
-      map.add("      PORT MAP ( " 
-          + LedArrayGenericHDLGeneratorFactory.LedArrayColumnAddress 
+      map.add("      PORT MAP ( "
+          + LedArrayGenericHDLGeneratorFactory.LedArrayColumnAddress
           + " => "
-          + LedArrayGenericHDLGeneratorFactory.LedArrayColumnAddress 
+          + LedArrayGenericHDLGeneratorFactory.LedArrayColumnAddress
           + id
           + ",");
       map.add("                 "
```

Comment:
In line with the other changes, should the .. + .. + .. + .. not be changed?

Reply:
Done

Reply:
Done